### PR TITLE
max and min marker-size range fix

### PR DESCRIPTION
**Pull request by michellemho**
This PR allows the size parameters "max" and "min" to be set. Before, the defaults would override user input. Now, user inputs override defaults.

### CODE REVIEW

File: cartoframes/layer.py [hunk 1]
```diff
@@ -286,15 +285,20 @@ def __init__(self, query, time=None, color=None, size=None,
...
+            # Default size range, bins, and bin_method
             size = {
                 'range': [5, 25],
-                'bins': 10,
+                'bins': 5,
                 'bin_method': BinMethod.quantiles,
             }
+            # Assign default range and update if min/max given
+            old_size['range'] = size['range']
+            if 'min' in old_size.keys():
```
Comment (andy-esch) [inline cartoframes/layer.py]: You can do `if 'min' in old_size` instead of using the `keys()` method.

File: cartoframes/layer.py [hunk 2]
```diff
@@ -286,15 +285,20 @@ def __init__(self, query, time=None, color=None, size=None,
...
+                old_size['range'][0] = old_size['min']
+            if 'max' in old_size.keys():
+                old_size['range'][1] = old_size['max']
+            # Update all the keys in size if they exist in old_size
             size.update(old_size)
-            # Since we're accessing min/max, convert range into a list
-            size['range'] = list(size['range'])
-            self.style_cols[size['column']] = None
```
Comment (andy-esch) [inline cartoframes/layer.py]: Why was this line removed? It's needed for validation of cartocss later in the `_setup` method.

Comment (michellemho) [reply cartoframes/layer.py]: Ah! That's probably a mistake.

File: cartoframes/layer.py [hunk 3]
```diff
@@ -286,14 +285,20 @@ def __init__(self, query, time=None, color=None, size=None,
...
             old_size = size
+            # Default size range, bins, and bin_method
             size = {
                 'range': [5, 25],
-                'bins': 10,
+                'bins': 5,
                 'bin_method': BinMethod.quantiles,
             }
+            # Assign default range and update if min/max given
+            old_size['range'] = size['range']
```
Comment (andy-esch) [inline cartoframes/layer.py]: This overwrites the requested size range. Updating to:
```python
old_size['range'] = old_size.get('range', size['range'])
```

**Pull request merged**